Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+   <head>
+      <title></title>
+      <link rel="stylesheet" type="text/css" href="custom.css">
+   </head>
+   <body>
+      <div>
+      <div class="leftSection">
+         <ul>
+            <li><a id="one" href="#First">First</a></li>
+            <li><a id="two" href="#Second">Second</a></li>
+            <li><a id="three" href="#Third">Third</a></li>
+            <li class="clear"></li>
```

Comment:
Create and add clearfix class to `<ul>`, remove empty `<li>`

Reply:
Done